src/ngraph_capture_variables.cc

```diff
@@ -28,25 +28,32 @@ namespace tensorflow {
 namespace ngraph_bridge {
 
 //
-// Utility function to check if placement on the NGRAPH device has been
-// requested.
-//
-// FIXME(amprocte): stubbed out for now because NGRAPH device is gone.
-//
-static bool NGraphPlacementRequested(const Node* node) { return true; }
+// Utility function to check if it is an output node
+// Skip capturing it, if yes.
+static bool CheckIfOutputNode(const Node* node,
+                              std::vector<string> skip_these_nodes) {
+  for (string& f : skip_these_nodes) {
+    if (node->name() == f) {
+      NGRAPH_VLOG(5) << "Found Output Node: " << node->name()
+                     << " - skip capturing it";
+      return false;
+    }
+  }
+  return true;
```

Comment:
we could use find:
```
bool found = std::find (skip_these_nodes.begin(), skip_these_nodes.end(), node->name()) != skip_these_nodes.end();
if (!found){
    //VLOG
}
return found;
```

Once we update to set, you could use set's find instead of the generic find